README.md

```diff
@@ -62,13 +62,24 @@ For more configuration options see [here](../../wiki/Configuration).
 
 Need more details? You can find more detailed installation guides for [Unix](../../wiki/Unix) (including your Raspberry Pi), [macOS](../../wiki/macOS), and [Windows](../../wiki/Windows).
 
-#### Building via Docker
-
-*Note*: Using this method, the bot will lose all sounds when restarting because the Docker container has its own file system.
-
-+ Simply clone the repo and run `docker build -t soundbot .` inside the folder.
-+ Afterwards start the bot via `docker run soundbot`.
-+ To run the container in the background use `docker run -d soundbot`.
+#### Building/Running via Docker
+
++ First, clone the repo and run `docker build -t soundbot .` inside the folder.
++ Make a separate directory where your soundbot's data will live (Ex. `/etc/discord-soundbot`)
++ Then create a `docker-compose.yml` file in that directory with the following contents:
+```yaml
```

Comment:
Can't we just commit the `docker-compose.yml`, let the user create the `config/config.json` as normal, and use that for the volumes? Than we wouldn't even need the sentence above this line, as the sounds folder is already there as well.